chapters/annotations.tex

```diff
@@ -817,9 +817,17 @@ \subsubsection{Text}\doublelabel{text}
 outline). The contents inherited from \lstinline!FilledShape! is deprecated, but kept for compatibility reasons.
 
 There are a number of common macros that can be used in the text, and
-they should be replaced when displaying the text as follows:
+they should be replaced when displaying the text as follows (in order such that the earliest ones have precedence,
+and using the longest sequence of identifier characters - alphanumeric and \_):
 
 \begin{itemize}
+\item
+  \%\% replaced by \%
+\item
+  \%name replaced by the name of the component (i.e. the identifier for
+  it in the enclosing class).
+\item
+  \%class replaced by the name of the class (only the last part of the hierarchical name).
 \item
   \%\emph{par} and \%\{\emph{par\}} replaced by the value of the
```

Comment:
Macro `par` is emphasized, but `name` and `class` are not. Seems arbitrary to me.

Reply:
There is a logical reason: "par" is a meta-symbol - meaning any parameters; whereas name and class must be written like that.